english/staypuft/ghostbusters/watch-ghostbusters/hire-lawyer/hire-lawyer.md

```diff
@@ -0,0 +1 @@
+It is no surprise that Columbia Pictures decides that Udacity and all contributers to this repository have broken several copyright laws when publishing unauthorized content about Ghostbusters on a public domain, in this case "GitHub", and sends a cease and desist letter to this effect.  After all you have been through, including being inside a giant sentient marshmallow and nearly being run over by a cab, you now face the potential for financial ruin.  Luckily you realize you are broke, leaving the expenses of hiring an attorney to Udacity and GitHub, who hopefully have a boatload of $$!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.